src/utils/projectHelpers.js

```diff
@@ -0,0 +1,7 @@
+export const isOwner = (user, project) => {
+  return (
+    user && user.profile && 
+    (user.profile.user === project.user_id || !project.identifier)
+  )
+}
+  
```

Comment:
Is it worth having a few noddy tests for this?

Reply:
done